README.md

```diff
@@ -1,26 +1,70 @@
-# apollo-link-rest
+---
+title: REST Link
+---
 
-## Problem to solve
-You have an existing REST set of services that you have worked on for a long time. However the limitations of it are piling up and you think GraphQL is a great solution to make your application faster and easier to develop. With `apollo-link-rest`, you can start trying out GraphQL without a full server. You can use it to prototype and even use it for third party services that don't yet have a GraphQL endpoint.
+## ⚠️ This library is under active development ⚠️
 
-Apollo Link Rest lets you query traditional REST API endpoints while writing GraphQL. It is designed to work with Apollo Client, but can even be used on its own with Apollo Link.
+This library is under active development. For information on progress check out
+[this issue](https://github.com/apollographql/apollo-link-rest/issues/3) or the
```

Comment:
The plan of action issue (#3) was out of date / based on the [contributor meeting](./designs/meetings/17-12-14.md) we decided to close that ticket, and break out the remaining tasks into new GitHub issues. Mind changing that link to: https://github.com/apollographql/apollo-link-rest/issues ?